sites/next.skeleton.dev/src/lib/generateLLM.ts

```diff
@@ -0,0 +1,326 @@
+		designContent += content + '\n\n---\n\n';
+	}
+	designContent = `# Design\n\n` + designContent;
+	return designContent;
+}
+
+async function processTailwind(): Promise<string> {
+	const tailwindDocs = (await getCollection('docs')).filter((entry) => entry.slug.startsWith('tailwind/'));
+	let tailwindContent = '';
+	for (const doc of tailwindDocs) {
+		let content = doc.body;
+		content = await processPreviewBlocks(content, 'html');
+		content = `# ${doc.data.title}\n${doc.data.description}\n\n${content}`;
+		tailwindContent += content + '\n\n---\n\n';
+	}
+	tailwindContent = `# Tailwind\n\n` + tailwindContent;
+	return tailwindContent;
+}
+
+/**
+ * Processes Components documentation.
+ * Replaces the `/meta` slug suffix with the framework-specific one.
+ */
+async function processComponents(framework: Framework): Promise<string> {
+	const metaEntries = (await getCollection('docs')).filter((entry) => entry.slug.startsWith('components/') && entry.slug.endsWith('/meta'));
+	let componentsContent = '';
+	for (const metaEntry of metaEntries) {
+		const docSlug = metaEntry.slug.replace(/\/meta$/, `/${framework}`);
+		const docEntry = await getEntry('docs', docSlug);
+		if (!docEntry) continue;
+		let content = docEntry.body;
+		content = await processPreviewBlocks(content, framework);
+		content = await processApiTables(content, docEntry.slug);
+		content = `# ${metaEntry.data.title}\n${metaEntry.data.description}\n\n${content}`;
+		componentsContent += content + '\n\n---\n\n';
+	}
+	componentsContent = `# Components\n\n` + componentsContent;
+	return componentsContent;
+}
+
+/**
+ * Processes Integrations documentation.
+ * Replaces the `/meta` slug suffix with the framework-specific one.
+ */
+async function processIntegrations(framework: Framework): Promise<string> {
+	const metaEntries = (await getCollection('docs')).filter(
+		(entry) => entry.slug.startsWith('integrations/') && entry.slug.endsWith('/meta')
+	);
+	let integrationsContent = '';
+	for (const metaEntry of metaEntries) {
+		const docSlug = metaEntry.slug.replace(/\/meta$/, `/${framework}`);
+		const docEntry = await getEntry('docs', docSlug);
+		if (!docEntry) continue;
+		let content = docEntry.body;
+		content = await processPreviewBlocks(content, framework);
+		content = await processApiTables(content, docEntry.slug);
+		content = `# ${metaEntry.data.title}\n${metaEntry.data.description}\n\n${content}`;
+		integrationsContent += content + '\n\n---\n\n';
+	}
+	integrationsContent = `# Integrations\n\n` + integrationsContent;
+	return integrationsContent;
+}
+
+/**
+ * Generates the complete documentation content.
+ * @param framework The target framework ('svelte' or 'react').
+ */
+export async function generateDocumentation(framework: Framework): Promise<string> {
+	let content = `<system>
+This documentation provides a comprehensive reference for the Skeleton v3 UI framework, featuring ${framework === 'svelte' ? 'Svelte and SvelteKit' : 'React'} examples.
```

Comment:
Just leave it at `svelte` or `react`, our components are framework agnostic. Something like this:
```suggestion
This documentation provides a comprehensive reference for the Skeleton v3 UI framework, featuring ${toUpperCase(framework)} examples.
```